de.manuel_kehl.go-for-it.json

```diff
@@ -0,0 +1,42 @@
+{
+  "id": "de.manuel_kehl.go-for-it",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "go-for-it",
+  "rename-desktop-file": "go-for-it.desktop",
+  "rename-icon": "go-for-it",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc", "--socket=x11",
+    /* Wayland access */
+    "--socket=wayland",
+    /* Access user files */
+    "--filesystem=home",
+    /* Dconf access */
+    "--filesystem=xdg-run/dconf",
+    "--filesystem=~/.config/dconf:ro",
+    "--talk-name=ca.desrt.dconf",
+    "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+    /* DBus access */
+    "--own-name=de.manuel-kehl.go-for-it",
+    /* Notification access */
+    "--talk-name=org.freedesktop.Notifications"
+  ],
+  "modules": [
+    {
+      "name": "go-for-it",
+      "buildsystem": "cmake",
+      "builddir": "build",
+      "sources": [{
+        "type": "git",
+        "url": "https://github.com/mank319/Go-For-It.git",
+        "branch": "release_v1.6",
+        "commit": "5f319139ab978f4cee99612227896c5b7e7300f5"
+      }],
+      "post-install": [
+        "install -Dm644 COPYING /app/share/licenses/go-for-it/COPYING"
```

Comment:
I'm not sure this is actually needed... Is there anything that expects this file to be installed at that location?

Reply:
It isn't exposed by anything but us distributing licenses with everything probably isn't a bad idea.

Reply:
Yeah, I do think that's a good idea and wonder if we were following any spec for this location, so that we can standardize other apps to it in the future as well...

Reply:
I am all for having a standardized location. I have done this for all my applications since it is pretty much standard practice in distribution packaging.

Based on my understanding a lot of licenses actually require this for compliance. I therefore make it habit to include them.

Reply:
Fedora and Arch use `share/licenses/app-name`

Reply:
Indeed, that is why I picked it.

Reply:
I guess in that case it should be `share/licenses/de.manuel_kehl.go-for-it` then?
Eventually, if this becomes a standard location, it would probably be a good idea for the directories to get exported too.

Reply:
Eventually we might want to include licenses for bundled libraries too

Reply:
Just to clarify, you do want me to change it to `share/licenses/de.manuel_kehl.go-for-it`?

As for included libraries, I always include licenses for those too. But in this case there were none.

Reply:
Yeah, I would put it in that directory, if one day we decide to export it.